app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -99,4 +121,53 @@ public Retrofit getRetrofit() {
 
         return retrofit;
     }
+
+    /**
+     * Provides {@SecurePreferences} singleton instance to access shared preferences
+     *
+     * @return A single instance of {@SecurePreferences}
+     */
+    public static SharedPreferences getSharedPreferences() {
```

Comment:
Does it have to be static?

Reply:
Yes it has. 
getSharedPreferences() is a global endpoint for accessing application's shared preferences.
Since we will need to access preferences from different places within the app, it has to be static.